5001-6000/LIT5905FidalaHawarya.xml

```diff
@@ -8,6 +8,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <title xml:lang="gez" xml:id="t1">ፊደለ፡ ሐዋርያ፡</title>
                 <title corresp="#t1" type="normalized">Fidala ḥawāryā</title>
                 <title xml:id="t2" xml:lang="en">Apostle's Alphabet</title>
+                <title xml:id="t3" xml:lang="en">1 John 1–7</title>
```

Comment:
The First Epistle of John has 5 chapters according to my private Protestant Bible. If this refers to verses 1 to 7 of the first chapter, should it be 1 John 1:1-7? 
In LIT3509Epistle, the book is abbreviated "1 Joh."